### DISCUSSION TITLE
can`t start firmware

### DISCUSSION THREAD

**Discussion by tiretrak** [marcel-licence/esp32_midi_sampler#63] [Q&A]
hi. i have  esp32 audio kit v2.2 with ESP32-A1S (2974) . after burn program i have loop start:

`

ELF file SHA256: 0000000000000000

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x1f (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1344
load:0x40078000,len:13516
load:0x40080400,len:3604
entry 0x400805f0

Connect to ES8388 codec... 0x00: 0x05
0x01: 0x40
AdcCh1!
MixChAMPL!
ES8388 setup finished!
Reg 0x00 = 0x05
Reg 0x01 = 0x40
Reg 0x02 = 0x00
Reg 0x03 = 0x00
Reg 0x04 = 0x3c
Reg 0x05 = 0x00
Reg 0x06 = 0x00
Reg 0x07 = 0x7c
Reg 0x08 = 0x00
Reg 0x09 = 0x88
Reg 0x0a = 0x50
Reg 0x0b = 0x02
Reg 0x0c = 0x0c
Reg 0x0d = 0x02
Reg 0x0e = 0x30
Reg 0x0f = 0x20
Reg 0x10 = 0x00
Reg 0x11 = 0x00
Reg 0x12 = 0x16
Reg 0x13 = 0xb0
Reg 0x14 = 0x32
Reg 0x15 = 0x06
Reg 0x16 = 0x00
Reg 0x17 = 0x18
Reg 0x18 = 0x02
Reg 0x19 = 0x22
Reg 0x1a = 0x00
Reg 0x1b = 0x02
Reg 0x1c = 0x08
Reg 0x1d = 0x00
Reg 0x1e = 0x1f
Reg 0x1f = 0xf7
Reg 0x20 = 0xfd
Reg 0x21 = 0xff
Reg 0x22 = 0x1f
Reg 0x23 = 0xf7
Reg 0x24 = 0xfd
Reg 0x25 = 0xff
Reg 0x26 = 0x1b
Reg 0x27 = 0xb8
Reg 0x28 = 0x28
Reg 0x29 = 0x28
Reg 0x2a = 0xb8
Reg 0x2b = 0x80
Reg 0x2c = 0x00
Reg 0x2d = 0x00
Reg 0x2e = 0x1e
Reg 0x2f = 0x1e
Reg 0x30 = 0x1e
Reg 0x31 = 0x1e
Reg 0x32 = 0x00
Reg 0x33 = 0xaa
Reg 0x34 = 0xaa
rev: 15329, 15329
Reverb is ready!
Setup Serial2 with 31250 baud with rx: 21 only
Setup MidiPort2 using Serial2
Total PSRAM: 4192091
Free PSRAM: 4147467
not enough PSRAM memory for sampleStorage!
Total PSRAM: 4192091
Free PSRAM: 4147467
sampleStorageLen: 2073733
sampleStorageLen: 47.02s
ESP.getFreeHeap() 144456
ESP.getMinFreeHeap() 144276
ESP.getHeapSize() 177216
ESP.getMaxAllocHeap() 110580
Total heap: 177216
Free heap: 144456
Total PSRAM: 4192091
Free PSRAM: 4147467
Firmware started successfully
Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.

Core  1 register dump:
PC      : 0x400d4cd7  PS      : 0x00060c30  A0      : 0x800d4fc9  A1      : 0x3ffdba30  
A2      : 0x3f800000  A3      : 0x3ffc4c88  A4      : 0x00000040  A5      : 0x3ffc4c88  
A6      : 0x3ffc4d88  A7      : 0x3ffc3c50  A8      : 0x3ffc3b10  A9      : 0x00000000  
A10     : 0x00000000  A11     : 0x3ffc4c88  A12     : 0x3ffc4d88  A13     : 0x00000000  
A14     : 0x00000003  A15     : 0x00060023  SAR     : 0x0000000a  EXCCAUSE: 0x0000001c  
EXCVADDR: 0x00000000  LBEG    : 0x4008641d  LEND    : 0x40086425  LCOUNT  : 0x00000027  

Backtrace:0x400d4cd4:0x3ffdba300x400d4fc6:0x3ffdba70 0x400d51ab:0x3ffdbac0 0x400da549:0x3ffdbae0 

ELF file SHA256: 0000000000000000

Rebooting...
ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x1f (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1344
load:0x40078000,len:13516
load:0x40080400,len:3604
entry 0x400805f0
`
any ideas ?

**Comment by tiretrak** [comment]
![11111111111](https://user-images.githubusercontent.com/7010744/179548652-8a4905d9-3042-4b0e-8ba4-37d14f697b24.png)

**Reply by marcel-licence** [reply]
I've updated the code. For some unknown reason the latest ESP32 board library does not support the allocation of the complete PSRAM.
Please update your code and try it again.
![grafik](https://user-images.githubusercontent.com/65245426/180610702-c8b62f4d-4fc9-4708-9173-912f5645d7bd.png)
